zettarepl/transport/base_ssh.py

```diff
@@ -29,8 +29,7 @@ def run(self):
         self._copy_stdout_from(self.stdout_fd)
 
     def wait(self):
-        self.logger.debug("Waiting for exit status")
-        exitcode = self.stdout_fd.channel.recv_exit_status()
+        self.logger.debug("Reading stdout")
```

Comment:
How about stderr? Is it possible or are you always piping stderr to stdout?

Reply:
Yes, stderr is piped to stdout
https://github.com/freenas/zettarepl/blob/4b1b62b3bdf13bfaf8d6474b500dc54498f6602c/zettarepl/transport/base_ssh.py#L28